Documentation/CRDs/Cluster/external-cluster.md

```diff
@@ -80,7 +89,8 @@ In order to configure an external Ceph cluster with Rook, we need to extract som
     ```
 
 !!! note
-    An existing non-restricted user cannot be downgraded to a restricted user by upgrading. Admin needs to create a new restricted user for this by re-running the script.The upgrade flag should only be used to append new permissions to users. It shouldn't be used for changing a csi user already applied permissions. For example, you shouldn't change the pool(s) a user has access to.
+    An existing non-restricted user cannot be converted to a restricted user by upgrading.
+    The upgrade flag should only be used to append new permissions to users. It shouldn't be used for changing a csi user already applied permissions. For example, you shouldn't change the pool(s) a user has access to.
 
 2. Copy the bash output.
```

Comment:
Step 1 (above on line 31), and step 2 here should really be headers. There is a lot in each of the sections.
```suggestion
### 2. Copy the bash output.
```